substrate/frame/support/src/traits/voting.rs

```diff
@@ -25,14 +25,21 @@ use sp_runtime::{traits::Member, DispatchError};
 use sp_std::prelude::*;
 
 pub trait VoteTally<Votes, Class> {
+	/// Initializes a new tally
 	fn new(_: Class) -> Self;
+	/// Returns the amount of positive votes for the tally
 	fn ayes(&self, class: Class) -> Votes;
+	/// Retrieves the ratio of positive votes for the tally, without multipliers (i.e. conviction, ranks, etc.)
```

Comment:
Ratio of positive votes to what? I would also say "returns" instead of "retrieves".